app/src/main/java/com/strv/movies/network/NetworkModule.kt

```diff
@@ -15,17 +17,28 @@ import retrofit2.Retrofit
 import retrofit2.converter.moshi.MoshiConverterFactory
 import retrofit2.create
 import java.util.concurrent.TimeUnit
+import javax.inject.Qualifier
 import javax.inject.Singleton
 
-@Module
-@InstallIn(SingletonComponent::class)
-object ApiModule {
+@Qualifier
+@MustBeDocumented
+@Retention(AnnotationRetention.RUNTIME)
+annotation class UnauthorizedRetrofitOkHttpClient
```

Comment:
Maybe just: AuthorizedOkHttpClient, UnauthorizedOkHttpClient, AuthorizedRetrofit, UnauthorizedRetrofit 🤔

Reply:
Yeah, that's better